src/js/view/controls/templates/nextup.ts

```diff
@@ -1,9 +1,12 @@
 import type { HTMLTemplateString } from 'types/generic.type';
+import { Browser } from 'environment/environment';
 
 export default (header = '', title = '', duration = '', closeAriaLabel = ''): HTMLTemplateString => {
+    let firefoxPipFix = Browser.firefox ? ' jw-nextup-firefox-pip-fix' : '';
```

Comment:
Might as well use const here since we don't redefine this variable.
```suggestion
    const firefoxPipFix = Browser.firefox ? ' jw-nextup-firefox-pip-fix' : '';
```